### ISSUE TITLE
inconsistencies in PreFreeSurferPipeline.sh with AnatomicalAverage.sh

### ISSUE THREAD

**Issue by aureliocarnero** [Washington-University/HCPpipelines#313]
In AnatomicalAverage.sh from PreFreeSurfer. The input Arguments have changed from -n, -o, -s, -m... to: --output, --image-list....  But this is not reflected in PreFreeSurferPipeline.sh which still expect the old format

**Comment by coalsont** [comment]
I have committed e1eafb1372cb65ba163a4896299c1a408a1069fa to master to try to fix this, give it a try.

**Comment by aureliocarnero** [comment]
it worked, thanks. But now I found something else along the the PreFreeSurferPipeline.sh.
When running robustfov (line 86), before in v4.7.0 it didn't complaint (I don't know if it was executed or not) but now, it tells the brain size is not given correctly as an argument for robustfov. 
To correct this, I just add the argument -b before the brain size as:

$FSLDIR/bin/robustfov -i ${fn}_reorient -r ${fn}_roi -m ${fn}_roi2orig.mat **-b** $BrainSizeOpt

**Comment by glasserm** [comment]
Perhaps the FSL usage changed since this was implemented.  We can fix it, but it will break old FSL versions.

Matt.

**Comment by aureliocarnero** [comment]
That's true. Better not to touch anything. Therefore I'm closing the issue as resolved. If I encounter any other inconsistency in the pipeline while executing it, I will open a new issue.
Thank you

**Comment by glasserm** [comment]
Well, we do need the pipelines to work with newer FSLs, the problem is we don't know in which version this changed, which makes it hard to maintain backwards compatibility.

**Comment by aureliocarnero** [comment]
True, let me find out in which fsl version happened
